openfecwebapp/templates/macros/legal.html

```diff
@@ -1,3 +1,28 @@
+{% macro filter(title, result_type, query) %}
+<div id="filters" class="filters filters--fixed is-open">
+  <div class="filters__hider">
```

Comment:
You don't need `.filters__hider` (which was renamed `.filters__content` in https://github.com/18F/fec-style/pull/446/ ) because these are always open. That div is just for hiding them.